File: AsthmaAlertClientNew/app/models/dust-dander.js
import DS from 'ember-data';

export default DS.Model.extend({
    date: DS.attr("string"),
    asthmaValue: DS.attr("number"),
    asthmaCategory: DS.attr("string"),
    dustDanderValue: DS.attr("number"),
    dustDanderCategory: DS.attr("string")
});

File: AsthmaAlertApi/Models/DustDander.cs
using JsonApiDotNetCore.Models;

namespace AsthmaAlertApi.Models
{
    public class DustDander : Identifiable
    {
        [Attr("date")]
        public string Date { get; set; }

        [Attr("asthma-value")]
        public int AsthmaValue { get; set; }

        [Attr("asthma-category")]
        public string AsthmaCategory { get; set; }

        [Attr("dust-dander-value")]
        public int DustDanderValue { get; set; }

        [Attr("dust-dander-category")]
        public string DustDanderCategory { get; set; }
    }
}